packages/patternfly-react/src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -31,13 +36,19 @@ FieldLevelHelp.propTypes = {
   /** additional fieldlevelhelp classes */
   content: PropTypes.node,
   /** leave popover/tooltip open  */
-  close: PropTypes.string,
+  rootClose: PropTypes.bool,
+  /** overlay placement */
+  placement: PropTypes.string,
+  /** button class */
+  buttonClass: PropTypes.string,
   /** children nodes  */
   children: PropTypes.node
 };
 FieldLevelHelp.defaultProps = {
   content: null,
-  close: 'true',
+  rootClose: true,
+  placement: 'top',
+  buttonClass: 'popover-pf-info',
```

Comment:
should we be adding additional classes rather than replacing `popover-pf-info`?

Reply:
works for me!